bentoml/_internal/yatai_client/__init__.py

```diff
@@ -104,28 +108,57 @@ def read(*args, **kwargs):
             raise KeyError("Can only wrap read/write methods")
 
 
+# Just make type checker happy
+class BinaryIOCast(io.BytesIO):
+    def __init__(self, *args: t.Any, **kwargs: t.Any) -> None:
+        super().__init__()
+
+
+CallbackIOWrapper: t.Type[BinaryIOCast] = t.cast(
+    t.Type[BinaryIOCast], _CallbackIOWrapper
+)
+
+
+# Just make type checker happy
+class ProgressCast(Progress):
```

Comment:
use protocol here?

Reply:
@bojiang Do not need `t.Protocol`, this dummy class is just to fix the type annotation error of the rich library, not to add type to it.

Reply:
okay